### PR TITLE
Feature/#97 - create unit tests for InputParser class 

### LINKED ISSUE #97: create unit tests for InputParser class

### PR DESCRIPTION
**Pull request by barbosamaatheus**
1. Added unit tests for the **InputParser** class
2. Added check for non-precessed input files

Resolves #97

### CODE REVIEW

File: src/main/arguments/InputParser.groovy [hunk 1]
```diff
@@ -20,16 +21,17 @@ class InputParser {
...
                 String name = line["name"]
-                
+
                 projectList.add(new Project(name, path))
             } else {
                 projectList.add(new Project(path))
             }
         }
-
+        if(projectList.size() == 0) throw new InvalidArgsException('The input file cannot be processed, the project list will be considered as empty')
```
Comment (rafaelmotaalves) [inline src/main/arguments/InputParser.groovy:30]: This exception stops execution right? In this case, I don't think the second part of the message is needed. Could be just 

```java
throw new InvalidArgsException('The input file cannot be processed')
```

Comment (rafaelmotaalves) [inline src/main/arguments/InputParser.groovy:30]: This exception stops execution right? In this case, I don't think the second part of the message is needed. Could be just

throw new InvalidArgsException('The input file cannot be processed')

Comment (pauloborba) [reply src/main/arguments/InputParser.groovy:30]: @barbosamaatheus?

Comment (barbosamaatheus) [reply src/main/arguments/InputParser.groovy]: done!

**Pull request merged**